normandy/control/static/control/js/routes.js

```diff
@@ -22,7 +22,7 @@ export default (
       component={RecipeForm}
       pageTitle="Edit Recipe"
       ctaButtons={[
-        {text: 'Preview', icon: 'eye', link: 'preview/'},
+        {text: 'Preview', icon: 'eye', link: 'preview/?testing=true'},
```

Comment:
Why add this back in? The idea was that the testing argument should completely replace this.